scripts/dev/Dockerfile

```diff
@@ -6,112 +6,152 @@ ENV NODE_VERSION 18.17.0
+  && rm "node-v$NODE_VERSION-linux-$ARCH.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt \
+  && apt-mark auto '.*' > /dev/null \
+  && find /usr/local -type f -executable -exec ldd '{}' ';' \
+  | awk '/=>/ { so = $(NF-1); if (index(so, "/usr/local/") == 1) { next }; gsub("^/(usr/)?", "", so); print so }' \
+  | sort -u \
+  | xargs -r dpkg-query --search \
+  | cut -d: -f1 \
+  | sort -u \
+  | xargs -r apt-mark manual \
+  && apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false \
+  && ln -s /usr/local/bin/node /usr/local/bin/nodejs \
+  # smoke tests
+  && node --version \
+  && npm --version
 
 RUN mkdir -p /app && \
-    mkdir -p /opt/hoop/sessions && \
-    apt-get update -y && \
-    apt-get install -y \
-        net-tools \
-        iproute2 \
-        xz-utils \
-        locales \
-        tini \
-        jq \
-        openssh-client \
-        apt-utils \
-        procps \
-        curl \
-        gnupg \
-        gettext-base \
-        gnupg2
+  mkdir -p /opt/hoop/sessions && \
+  apt-get update -y && \
+  apt-get install -y \
+  net-tools \
+  iproute2 \
+  xz-utils \
+  locales \
+  tini \
+  jq \
+  openssh-client \
+  apt-utils \
+  procps \
+  unzip \
+  curl \
+  gnupg \
+  gettext-base \
+  gnupg2 \
+  libaio1 \
+  wget
 
 RUN echo "deb http://apt.postgresql.org/pub/repos/apt/ focal-pgdg main" | tee /etc/apt/sources.list.d/pgdg.list && \
-    echo "deb [arch=amd64,arm64] https://repo.mongodb.org/apt/ubuntu focal/mongodb-org/5.0 multiverse" | tee /etc/apt/sources.list.d/mongodb-org-5.0.list && \
-    curl -sL https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - && \
-    curl -sL https://www.mongodb.org/static/pgp/server-5.0.asc | apt-key add -
+  echo "deb [arch=amd64,arm64] https://repo.mongodb.org/apt/ubuntu focal/mongodb-org/5.0 multiverse" | tee /etc/apt/sources.list.d/mongodb-org-5.0.list && \
+  curl -sL https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - && \
+  curl -sL https://www.mongodb.org/static/pgp/server-5.0.asc | apt-key add -
 
 RUN apt-get update -y && \
-    apt-get install -y \
-        openjdk-11-jre \
-        default-mysql-client \
-        postgresql-client-15 \
-        mongodb-mongosh mongodb-org-tools mongodb-org-shell mongocli
+  apt-get install -y \
+  openjdk-11-jre \
+  default-mysql-client \
+  postgresql-client-15 \
+  mongodb-mongosh mongodb-org-tools mongodb-org-shell mongocli
+
+# https://download.oracle.com/otn_software/linux/instantclient/1924000/instantclient-basic-linux.x64-19.24.0.0.0dbru.zip
+# https://download.oracle.com/otn_software/linux/instantclient/1924000/instantclient-sqlplus-linux.x64-19.24.0.0.0dbru.zip - x86-64
+
+# https://download.oracle.com/otn_software/linux/instantclient/1924000/instantclient-basic-linux-19.24.0.0.0dbru.zip
+# https://download.oracle.com/otn_software/linux/instantclient/1924000/instantclient-sqlplus-linux-19.24.0.0.0dbru.zip - x86
+
+
+# Baixar e instalar o Oracle Instant Client e SQL*Plus
```

Comment:
I do not oppose to comments, just pointing out because it's in Portuguese 😅

Reply:
Thx! I forgot this one